### DISCUSSION TITLE
Update node attributes using model reactitivity?

### DISCUSSION THREAD

**Discussion by yellow1912** [ueberdosis/tiptap#1558] [General Questions] [answered]
Hello,

I wonder if it's possible to do something like this
```

`
    <node-view-wrapper class="vue-component">
    <figure>
      <img v-bind:src="node.attrs.src" v-bind:alt="node.attrs.alt"/>
      <node-view-content as="figcaption"/>
      <input type="text" v-model="node.attrs.alt"/>
    </figure>
    </node-view-wrapper>`,
```

I'm using interactive node view (https://www.tiptap.dev/guide/node-views/vue) and I want to allow users to directly type in the alt text. Is it possible? I see that the alt is updated but it doesn't seem to be updated back to the node attrs for some reason.

**Comment by hanspagel** [comment, marked as answer]
Try using the updateNodeAttributes prop instead.

https://www.tiptap.dev/guide/node-views/vue#update-node-attributes

**Reply by yellow1912** [reply]
Thank you, I think it works but not sure if it is the correct way to do it, below is my code:

```
data() {
    return {
      alt: this.node.attrs.alt
    }
  },
  watch: {
    alt(newVal) {
      this.updateAttributes({alt: newVal});
    }
  }
```

My template is now:

```
`
    <node-view-wrapper class="vue-component">
    <figure>
      <img v-bind:src="node.attrs.src" v-bind:alt="node.attrs.alt"/>
      <node-view-content as="figcaption"/>
      <input type="text" v-model="alt"/>
    </figure>
    </node-view-wrapper>`
```
